### ISSUE TITLE
user settings - scheduled/manual refresh from within the settings dialog doesn't work properly

### ISSUE THREAD

**Issue by sgratch** [oVirt/ovirt-web-ui#1450]
If user settings dialog is open and no changes were done by the user but a property was changed not by the current browser (other browser or via rest) and a refresh is activated to fetch backend data (manual or scheduled)  ->
then instead of updating the dialog relevant fields based on backend values , the 'save' button becomes enabled. Only leaving and re-entering the settings dialog will update the fields as it should.

We should either block manual/scheduled refreshes while settings dialog is opened or just fetch all data from backend and update ui fields. if no changes were done on ui (so no conflicts). As it is now- it seems strange.

We can also consider adding a message so that the user will know and suggest him to re-enter the dialog.

**Comment by michalskrivanek** [comment]
it was discussed a long time ago - this is such a corner case it's really not worth the effort, the whole "background refresh" shouldn't have been implemented....

**Comment by sgratch** [comment]
@michalskrivanek Exactly, so I suggest to block the refresh while dialog is opened. Current scenario is that once one of the backend properties were changed, the 'save' button suddenly becomes enabled without a reason. The user won't know why and leaving the dialog requires applying that all user changes are discarded which is even more strange since nothing was triggered by the user on 1st place

**Comment by michalskrivanek** [comment]
i don't midn the current behavior either tbh. it will never happen in reality anyway.

**Comment by sgratch** [comment]
> i don't midn the current behavior either tbh. it will never happen in reality anyway.

OK, I agree it's a corner case but it's also easy to fix (if current page is settings -> skip refresh).
We are not planning on handling it for 4.4.7 anyway.
